### ISSUE TITLE
Listing destination project IDs as json output does not work

### ISSUE THREAD

**Issue by drewmullen** [hashicorp-services/tfm#115]
**Describe the bug**

```shell
$ tfm list projects --side destination --json
Getting list of projects from:  app.terraform.io
Found 10 Projects
╭──────────────────────────────────────┬──────────────────────╮
│ NAME                                 │ ID                   │
├──────────────────────────────────────┼──────────────────────┤
│ tfm-test                             │ prj-<> │
╰──────────────────────────────────────┴──────────────────────╯
```

I tried moving the flag to a few different places, same result as above:
```shell
$ tfm list --json projects --side destination
$ tfm --json list projects --side destination
```

**Works great with source workspaces:**
```shell
$ tfm list workspaces --json
$ tfm list workspaces --side destination --json
```

**Comment by drewmullen** [comment]
Also - Please when yall implement this, can you include the project id in the json output, not just the name. I've just noticed that `list workspaces --json` only outputs the workspace name. I would like to be able to programmatically use the list project --json to create mappings of the project name and its id

For now, `awk` seems to work mostly except for project names with spaces:

```shell
$ gawk '{print $2"="$4}' output.txt
<tfm-test>=prj-< >
Default=│
```

**Comment by drewmullen** [comment]
actually `aws` does work if you use the `|` as the FS but notice that the output is not a normal `|`. That took me a minute to figure out 😅 

`gawk 'BEGIN { FS="│" } {print $2"="$3}' output.txt`

**Comment by Josh-Tracy** [comment]
This feature is now available in the 0.4.0 release.
